common/src/thrift/worker.thrift

```diff
@@ -4,9 +4,16 @@ include "common.thrift"
 include "exception.thrift"
 
 service WorkerService extends common.TachyonService {
-  void accessBlock(1: i64 blockId)
 
-  bool asyncCheckpoint(1: i64 fileId)
+  /**
+   * access a block given the block id
+   */
+  void accessBlock( /** the id of the block being accessed */ 1: i64 blockId)
+
+  /**
+   * asynchronously checkpoint a file: return whether the checkpoint operation succeeded
```

Comment:
`Asynchronously checkpoints a file: return whether the checkpoint operation succeeded.`